### PR TITLE
LA-514 Search in upload request

### PR DESCRIPTION
**Pull request by imGok**
#514 

search

### CODE REVIEW

**Comment by huynguyennovem** [comment]
The floating button should same as design: https://www.figma.com/file/sEBIXTIsTfTb1TKZvMnuGx/LinShare-iOS-App?node-id=2974%3A0

Current: 
drawing

**Comment by hoangdat** [comment]
> The floating button should same as design: https://www.figma.com/file/sEBIXTIsTfTb1TKZvMnuGx/LinShare-iOS-App?node-id=2974%3A0
> 
> Current:
> drawing

IMO, in the search screen, we don't need the `Add button`.

**Comment by huynguyennovem** [comment]
> > The floating button should same as design: https://www.figma.com/file/sEBIXTIsTfTb1TKZvMnuGx/LinShare-iOS-App?node-id=2974%3A0
> > Current:
> > drawing
> 
> IMO, in the search screen, we don't need the `Add button`.

It's button in the Upload Request Group screen. We need it order to navigate into Create Upload Request

File: lib/presentation/view/background_widgets/background_widget_builder.dart [hunk 1]
```diff
@@ -54,26 +54,24 @@ class BackgroundWidgetBuilder {
...
-                textAlign: TextAlign.center,
-              ),
-            ),
-          ],
-        )),
-        ListView()
-      ]);
+    return Center(
+        key: _key ?? Key('BackgroundWidgetBuilder'),
+        child: SingleChildScrollView(
```
Comment (huynguyennovem) [inline lib/presentation/view/background_widgets/background_widget_builder.dart:59]: Why do we need a `SingleChildScrollView` here?

Comment (imGok) [reply lib/presentation/view/background_widgets/background_widget_builder.dart:59]: I need it to fix a bottom overflow when we return from searching

**Review by hoangdat — APPROVED** [review]

**Comment by huynguyennovem** [comment]
look good to me

@imGok  pls resolve conflict

**Review by hoangdat — APPROVED** [review]

**Pull request merged**